vignettes/ArrayExpress-example.Rmd

```diff
@@ -223,10 +222,20 @@ We associate active signatures to known KEGG pathways.
 
 ```{r}
 pathway_result <- annotate_signatures_with_genesets(
-  selected_signatures = active_sigs, model = model, genesets = KEGG)
+  selected_signatures = unique_active_sigs, model = model, genesets = KEGG)
 DT::datatable(pathway_result)
 ```
 
+In this case, there are also signatures uncharacterized by KEGG.
+```{r}
+uncharacterized_sigs <- setdiff(unique_active_sigs, pathway_result$signature)
+uncharacterized_sigs
+```
+
+Check the signature similarity of the uncharacterized signatures.
+```{r}
+plot_signature_overlap(selected_signatures = uncharacterized_sigs)
```

Comment:
Any reason why this plot is not saved to a variable, but it is in [line 162](https://github.com/greenelab/ADAGEpath/pull/17/commits/339c4de1d7728ab6014c4f656964efebead6b151#diff-5d0e61e846b3ebc3c7f7a74b273574e2R162)